app/views/pageflow/entries/show.html.erb

```diff
@@ -11,6 +11,7 @@
 
     <%= tag :link, :rel => 'icon', :href => image_path("pageflow/themes/#{@entry.theme.directory_name}/favicon.ico"), :type => 'image/ico' %>
     <%= meta_tags_for_entry(@entry) %>
+    <%= auto_discovery_link_tag(:atom, entries_url(format: 'atom')) %>
```

Comment:
This links to the global, unscoped entries feed, right?

Reply:
Nope, it uses the hostname of the Entry.

That being said, I should not include this tag for stories not running under a custom CNAME.